scale.c

```diff
@@ -28,8 +28,12 @@
 #endif
 
 #ifndef M_PI
+#ifndef PI
+#define M_PI 3.14159265358979323846
```

Comment:
I recall running in to this issue on windows in the past as well.  Apparently defining math constants in math.h is not strictly ANSI C, so MS doesn't include them by default.  Defining _USE_MATH_DEFINES before including math.h will avoid the need to manually define it with Visual C++ though.

Reply:
Fixed the issue with your recommendation.